ocs_ci/ocs/cluster.py

```diff
@@ -903,6 +904,88 @@ def get_osd_utilization():
     return osd_filled
 
 
+def get_ceph_df_detail():
+    """
+    Get ceph osd df detail
+
+    Returns:
+         dict: 'ceph df details' command output
+         
+    """
+    ceph_cmd="ceph df detail"
+    ct_pod = pod.get_ceph_tools_pod()
+    output = ct_pod.exec_ceph_cmd(ceph_cmd=ceph_cmd)
+    return output
+
```

Comment:
you can change:
return ct_pod.exec_ceph_cmd(ceph_cmd=ceph_cmd)

Reply:
Changed